### ISSUE TITLE
Windows 10 Pro crash

### ISSUE THREAD

**Issue by TheyCantHandle** [jawj/IKEv2-setup#39]
I followed the guide carefully, everything was then working well. I was connected to my VPN (connection established). So I tested browsing on www, it worked great but few minutes after my PC crashes (bluescreen) giving an error and I tried it again and again and everytime few minutes after I start the connection, windows 10 crashes.

**Comment by jawj** [comment]
I have not come across this, and it seems kind of unlikely this has anything to do with the VPN setup defined here — I would think this is either a bug in the Windows 10 VPN or perhaps more likely a problem with your PC?

**Comment by TheyCantHandle** [comment]
Yeah since yesterday my PC is trying to install an update without any success so maybe this is related. I had another issue though but I think this is more related to vpns overall. How do websites detect usage of vpn? Netflix detects usage of my own vpn.

**Comment by jawj** [comment]
I think they do it simply by IP address. They will pretty certainly figure it out if you are using AWS, Google, Digital Ocean or any other large provider for your VPN. The best bet is to pick a small and obscure VPS provider in the hope that their IP range is not on any blacklist.
